containers/podman-desktop-extension-bootc
# chore: deselect all other options when selecting iso

chore: deselect all other options when selecting iso

### What does this PR do?

* Disables all other options when selecting ISO
* Shows a warning
* Requires
  https://github.com/containers/podman-desktop-extension-bootc/pull/803
  in first

### Screenshot / video of UI




https://github.com/user-attachments/assets/bba03d9c-5f38-40f8-9ea7-0a490a175a0b



### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop-extension-bootc/issues/702

### How to test this PR?



1. Select iso option when building
2. See that it disables everything else

Signed-off-by: Charlie Drage <charlie@charliedrage.com>
